ts/Core/Chart/Chart.ts

```diff
@@ -2405,20 +2403,81 @@ class Chart {
         const tempWidth = chart.plotWidth;
 
         axes.some(function (axis: Axis): (boolean|undefined) {
```

Comment:
Since the calculation of the `expectedSpace` are more elaborate now maybe instead of `axes.some` there should be done a search for a max `expectedSpace` across all axes? There might be more than 1 axis that will cause `expectedSpace` to be calculated and that 1 might not produce the biggest value causing a redo or unstable render.

Reply:
I am not sure if we should address this issue at that moment. After my investigation, calculating `expectedSpace` for multiple axes is really complicated and probably will need some time/performance to calculate everything correctly. Also, I think, if we would like to calculate it, `expectedSpace` should be the sum of all expected spaces for every axis, but keeping in mind options like `axis.offset`/`axis.opposite`. Now it is handled by the additional trigger of recalculation (`redoVertical`).